protocol/0081-SUCM-successor_markets.md

```diff
@@ -104,6 +106,14 @@ When fetching a market that is part of a "parent / successor chain", we should s
 
 After a LNL checkpoint restart the successor (child) / parent market state is preserved where applicable inc. the LPs ELS	(<a name="0081-SUCM-016" href="#0081-SUCM-016">0081-SUCM-016</a>)
 
+A market which has been settled, but is still inside successor expiry window, is retained in a checkpoint, and can be used by a successor market after restart(<a name="0081-SUCM-029" href="#0081-SUCM-029">0081-SUCM-029</a>)
+
+A market which has been settled, and beyond the successor expiry window, is not retained in a checkpoint, and cannot be used by a successor market after restart(<a name="0081-SUCM-030" href="#0081-SUCM-030">0081-SUCM-030</a>)
+
+A market which has been settled, and already has a child which has succeeded it, is retained in a checkpoint, and can be found after restart. Both child and parent retain child/parent links in the API(<a name="0081-SUCM-031" href="#0081-SUCM-031">0081-SUCM-031</a>)
```

Comment:
> A market which has been settled, and already has a child which has succeeded it, is retained in a checkpoint, and can be found after restart

Aren't there two cases? 
1. Parent has settled but window for starting a successor market hasn't yet passed. In that case the child can (at the 1st ending of opening auction) allow LPs to reclaim their virtual stakes. 
2. Parent has settled and the window for starting a successor has passed. In that case we don't expect to see the parent stored in LNL checkpoint.

Reply:
Aha, I see that case 2. is in 030; so just clarify 031.